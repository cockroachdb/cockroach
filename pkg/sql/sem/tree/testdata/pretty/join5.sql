select * from
  (a cross join b),
  (a cross join (c cross join d)),
  d,
  (table x),
  (table e order by f)
