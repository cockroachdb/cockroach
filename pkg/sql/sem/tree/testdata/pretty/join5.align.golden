1:
-
SELECT
	*
FROM
	a
	CROSS JOIN
		b,
	a
	CROSS JOIN
		(
			c
			CROSS JOIN
				d
		),
	d,
	(
		TABLE
			x
	),
	(
		TABLE
			e
		ORDER BY
			f
	)

6:
------
SELECT
	*
FROM a
     CROSS JOIN
		b,
     a
     CROSS JOIN
		(
			c
			CROSS JOIN
				d
		),
     d,
     (
		TABLE
			x
     ),
     (
		TABLE
			e
		ORDER BY
			f
     )

8:
--------
SELECT *
FROM a
     CROSS JOIN
		b,
     a
     CROSS JOIN
		(
			c
			CROSS JOIN
				d
		),
     d,
     (
		TABLE
			x
     ),
     (
		TABLE
			e
		ORDER BY
			f
     )

15:
---------------
SELECT *
FROM a
     CROSS JOIN
		b,
     a
     CROSS JOIN
		(
			c
			CROSS JOIN
				d
		),
     d,
     (TABLE x),
     (
		TABLE e
		ORDER BY
			f
     )

17:
-----------------
SELECT *
FROM a
     CROSS JOIN
		b,
     a
     CROSS JOIN (
					c
					CROSS JOIN
						d
                ),
     d,
     (TABLE x),
     (
		TABLE e
		ORDER BY
			f
     )

18:
------------------
SELECT *
FROM a
     CROSS JOIN b,
     a
     CROSS JOIN (
					c
					CROSS JOIN
						d
                ),
     d,
     (TABLE x),
     (
		TABLE e
		ORDER BY f
     )

25:
-------------------------
SELECT *
FROM a
     CROSS JOIN b,
     a
     CROSS JOIN (
					c
					CROSS JOIN
						d
                ),
     d,
     (TABLE x),
     (TABLE e ORDER BY f)

32:
--------------------------------
SELECT *
FROM a
     CROSS JOIN b,
     a
     CROSS JOIN (
					c
					CROSS JOIN d
                ),
     d,
     (TABLE x),
     (TABLE e ORDER BY f)

33:
---------------------------------
SELECT *
FROM a
     CROSS JOIN b,
     a
     CROSS JOIN (c CROSS JOIN d),
     d,
     (TABLE x),
     (TABLE e ORDER BY f)

86:
--------------------------------------------------------------------------------------
SELECT *
FROM a CROSS JOIN b, a CROSS JOIN (c CROSS JOIN d), d, (TABLE x), (TABLE e ORDER BY f)

95:
-----------------------------------------------------------------------------------------------
SELECT * FROM a CROSS JOIN b, a CROSS JOIN (c CROSS JOIN d), d, (TABLE x), (TABLE e ORDER BY f)
